samples/js/node/pronunciationAssessmentContinue.js

```diff
@@ -4,12 +4,12 @@
 // pull in the required packages.
 import * as sdk from "microsoft-cognitiveservices-speech-sdk";
 import * as filePushStream from "./filePushStream.js";
-import * as Segment from "segment";
```

Comment:
Have you removed the depency of Segment in https://github.com/Azure-Samples/cognitive-services-speech-sdk/blob/master/ThirdPartyNotices.md ?

Reply:
> Have you removed the depency of Segment in https://github.com/Azure-Samples/cognitive-services-speech-sdk/blob/master/ThirdPartyNotices.md ?

removed now